models/ccsr/ccsr_models.yml

```diff
@@ -30,12 +30,7 @@ models:
       - name: ccsr_category
         description: "A CCSR category for the ICD-10-CM code. One ICD-10-CM code may have up
           6 CCSR categories associated, ordinally ranked by the ccsr_category_rank.
-          The not_null test will fail on this dimension if a record has a `code` that isn't
-          referenced in the CCSR seed file."
-        tests:
```

Comment:
Removed a bunch of unnecessary tests related to encounter_id from the CCSR mart. We can add these back when we create encounters for other types, not just acute_inpatient.